mod/wikirate_source/spec/set/type/source/source_spec.rb

```diff
@@ -70,19 +70,19 @@
       end
     end
     context "while creating with more than one source type " do
-      it do 
+      it do
         url = 'http://www.google.com/?q=wikirateissocoolandawesomeyouknow'
-        
+
         sourcepage = Card.new :type_id=>Card::SourceID,:subcards=>{ '+Link' => {:content=> url},'+File' =>{:type_id=>Card::FileID}, '+Text' => {:content=>"Hello boys!",:type_id=>Card::BasicID} }
         expect(sourcepage).not_to be_valid
         expect(sourcepage.errors).to have_key :source
         expect(sourcepage.errors[:source]).to include("Please only add one type of source")
       end
     end
-    describe "while creating a source with a file link" do 
+    describe "while creating a source with a file link" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.